index.html

```diff
@@ -489,12 +489,12 @@ <h2>Contact me</h2>
           </p>
         </div>
         <form action="https://formspree.io/f/xrgvogvr" method="POST" class="flex-column align-center">
-          <label for="name">Full name</label>
+          
           <input type="text" maxlength="30" name="name" placeholder="&nbsp;&nbsp;Full name" class="contact-input"
             required />
-          <label for="email">Email Address</label>
+          
           <input type="email" name="email" placeholder="&nbsp;&nbsp;Example@gmail.com" class="contact-input" required />
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that the color of the placeholders of the inputs much the Figma design as much as possible. This will enable you to have a better UI ☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/230740267-5e0a97d6-0bf8-473b-bd50-8c5db49b4372.png)

Figma design
![image](https://user-images.githubusercontent.com/99394374/230740280-8497ee41-7146-4003-8301-f832bca15c5f.png)